jvm-runtime/plugin/common/service.go

```diff
@@ -233,7 +231,12 @@ func (s *Service) runQuarkusDev(ctx context.Context, projectConfig projectconfig
 				Errors: ers,
 			}}}), nil
 	case err := <-errorChan:
```

Comment:
errChan now gets either the failure to launch error, or the error from the above `go func() {}` (eg connect timeout), whichever fails first